.devcontainer/postCreateCommand.sh

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env bash
 python -m pip install poetry
+python -m pip install poetry-dynamic-versioning
```

Comment:
Would it make sense to add directions to the contributing docs to make sure users who don't leverage devcontainers may still use this?

Reply:
Our contributing docs for manual setup instruct users to run `.devcontainer/postCreateCommand.sh` to finish setting their environment. So I don't think additional instructions are required.